src/openapi/streaming/connection/transport/websocket-transport.ts

```diff
@@ -107,6 +109,7 @@ class WebsocketTransport implements StreamingTransportInterface {
 
         this.failCallback = failCallback;
         this.inactivityFinderRunning = false;
+        this.reconnectOnceOnlineCallback = this.reconnect.bind(this);

```

Comment:
do we want to be straight up reconnecting, without even checking if we are still connected?
What about signal-r core - I thought that had the same issue?